performant-software/DM
# Preserve highlight functionality after list transformation

### What does this PR do?
Updates mouse event listeners for text highlights after a selection of text is turned into a bulleted or numbered list.

### What issues does it address?
Closes #133 

### How to test
- On [staging](http://dm-staging.middell.net/), open an editable text resource and add a highlight somewhere. 
- Select part of the text that includes the highlight.
- Click on the bullet list or numbered list button to transform the selected text.
- Hold the cursor over the highlight to confirm it still responds correctly (bringing up the annotations popup).
